### PR TITLE
packaging: fix policyutils dependency

### PR DESCRIPTION
**Pull request by safchain**
skip skydive-compile-tests
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-go-fmt
skip skydive-k8s-tests
skip skydive-scale-tests
skip skydive-selenium-tests
skip skydive-unit-tests
skip skydive-cdd-overview-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run packaging-tests

**Comment by safchain** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run vagrant-tests

**Comment by lebauce** [comment]
run vagrant-tests

**Comment by lebauce** [comment]
run vagrant-tests

File: scripts/ci/run-vagrant-tests.sh [hunk 1]
```diff
@@ -16,7 +16,11 @@ cd ${GOPATH}/src/github.com/skydive-project/skydive
 
 if [ "$DEVMODE" == "true" ]; then
     make static
-    make rpm
+    make srpm
+    VERSION=$(make -s version | cut -d '-' -f 1)
+    TAG=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
+    mock -r epel-7-x86_64 -D "fullver $(make -s version)" --resultdir rpmbuild/RPMS --rebuild rpmbuild/SRPMS/skydive-${VERSION}*${TAG}.src.rpm
```
Comment (nplanel) [inline scripts/ci/run-vagrant-tests.sh]: use centos-7-x86_64 target instead of epel-7  (default package build dependency)

Comment (nplanel) [reply scripts/ci/run-vagrant-tests.sh]: the full fix is
"""
    # mock centos/rhel package build as dependency are not the same (policycoreutils-python{,-utils})
    make srpm
    VERSION=$(make -s version | awk -F '-' '{print $1}')
    SHA1=$(make -s version | awk -F '-' '{print $2}')
    mock -r centos-7-x86_64 -D "fullver $(make -s version)" --rebuild rpmbuild/SRPMS/skydive-${VERSION}-*${SHA1}.src.rpm
    mkdir -p rpmbuild/RPMS/{x86_64,noarch}
    mv /var/lib/mock/centos-7-x86_64/result/skydive*-${VERSION}-*${SHA1}*.x86_64.rpm rpmbuild/RPMS/x86_64/
    mv /var/lib/mock/centos-7-x86_64/result/skydive*-${VERSION}-*${SHA1}*.noarch.rpm rpmbuild/RPMS/noarch/
"""

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

File: contrib/ansible/roles/skydive_common/tasks/package.yml [hunk 2]
```diff
@@ -19,17 +19,11 @@
     mode: 0755
   when: '"file://" in skydive_binary_remote_location'
 
-- name: Move skydive packages
-  shell: "mv /tmp/*/*.rpm /tmp"
+- name: Install skydive packages
```
[the remark this answers is not in the document]
Comment (lebauce) [reply contrib/ansible/roles/skydive_common/tasks/package.yml]: We did modify this when we decided to build the RPM and copy it into the VM. Why not this time ?

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

File: scripts/ci/run-vagrant-tests.sh (hunk 1 above)
Comment (lebauce) [reply scripts/ci/run-vagrant-tests.sh]: Was the centos-7-x86_64 manually copied on the CI ?

Comment (lebauce) [reply scripts/ci/run-vagrant-tests.sh]: SHA1=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
to handle the tainted case

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Pull request merged**